MailchimpSDK/MailchimpSDKTests/MailchimpSDKTests.swift

```diff
@@ -0,0 +1,326 @@
+//
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+
+// NOTE: Commented tests require a working API and API test token to pass. They are disabled until those prerequisites are met.
+
+import XCTest
+@testable import MailchimpSDK
+
+class MailchimpSDKTests: XCTestCase {
+
+    var token = "PUT YOUR API TOKEN HERE"
+    var timeout: TimeInterval = 10
+    
+    override func setUp() {
+        try? MailchimpSDK.initialize(token: token)
+    }
+    
+    func testContactRequest() {
+        let api = AnzeeAPI(token: token)
+        let contact: Contact = Contact(emailAddress: "test@mailchimp.com")
+        XCTAssertNotNil(contact)
+
+        let testExpectation = expectation(description: "Network Request")
+        let request = ContactRequest(contact: contact) { result in
+            switch result {
+            case .success(let responseData):
+                XCTAssertNotNil(responseData)
+            case .failure(let error):
+                XCTAssertNotNil(error as APIError)
+                if case .apiError(let response) = error {
+                    XCTAssertEqual(response.status, 404)
+                    XCTAssertEqual(response.type, "http://developer.mailchimp.com/documentation/mailchimp/guides/error-glossary/")
+                    XCTAssertEqual(response.detail, "Invalid path")
+                }
+            }
+            testExpectation.fulfill()
+        }
+        
+        api.process(request: request)
+        waitForExpectations(timeout: timeout, handler: nil)
+    }
+    
+    func testSDKInitialize() {
+        do {
+            try MailchimpSDK.initialize(token: "atesttoken-us1")
+        } catch {
+            XCTFail("MailchimpSDK cannot be initialized with the token: \(token)")
+        }
+        XCTAssertNotNil(MailchimpSDK.api)
+    }
+
+    /*
+    func testCreateContact() {
+        let contact: Contact = Contact(emailAddress: "test@mailchimp.com")
+
+        let testExpectation = expectation(description: "Network Request")
+        let dataTask = Audience.create(contact: contact) { result in
+            switch result {
+            case .success(let responseString):
+                XCTAssertNil(responseString) // API endpoint does not yet exist. When this test starts failing, we will update it.
+            case .failure(let error):
+                XCTAssertNotNil(error as APIError)
+                if case .apiError(let response) = error {
+                    XCTAssertEqual(response.status, 404)
+                    XCTAssertEqual(response.type, "http://developer.mailchimp.com/documentation/mailchimp/guides/error-glossary/")
+                    XCTAssertEqual(response.detail, "Invalid path")
+                }
+            }
+            testExpectation.fulfill()
+        }
+        XCTAssertNotNil(dataTask)
+        waitForExpectations(timeout: timeout, handler: nil)
+    }
```

Comment:
Should we keep these commented out tests?

Reply:
Removed. Thanks!